attributes/default.rb

```diff
@@ -24,28 +17,18 @@
 # Will be populated automaticaly when roles use, unless overrided.
 default['stackstorm']['components'] = %w(st2common)
 default['stackstorm']['service_binary'] = {}
+default['stackstorm']['install_stackstorm']['version'] = '1.4.0-8'
```

Comment:
> we follow automatic updates approach

I would add a warning label in the readme then if that is how you want versions handled here